src/utils/url.ts

```diff
@@ -1,79 +0,0 @@
-import urljoin from 'url-join';
```

Comment:
It should be possible to remove this dependency now.

Reply:
nice catch 🎯